### PR TITLE
Review request for Hello Microverse Project

### PR DESCRIPTION
**Pull request by petushka1**
Adding linters to the Project. Adding HTML and CSS files.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: gray;
+}
```
Comment (thecodechaser) [inline style.css]: - [ ] You are using an element selector to style the heading, Please use a class selector instead to follow the project requirements. You can add a class to the heading and use that class in the .css file to style the heading.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @petushka1👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- GitHub flow is followed. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
- [ ] Please create a `test.md` file with the text "This file should be ignored by git", and mention this filename `test.md` in `.gitignore` file, and the file will be ignored by git and will not push to Github.
- [ ] Your setup the wrong file for the linter. Please install the [linter ](https://github.com/microverseinc/linters-config) correctly and make sure that all linter tests are passing.
- [ ] Please add a descriptive README file, feel free to use this [Template](https://github.com/microverseinc/readme-template) but customize it for your project.
- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1 +1,25 @@
...
+
+
+## Getting Started
+
+**Setup HTML & CSS Linters in Workflows folder through Actions ob Github**
+**Add .gitignore file**
+**Add CSS and HTML files file**
+**Complete test and add test.md file**
+
+
```
Comment (chrystalme) [inline README.md]: Kindly add how to have a local copy of your project and run it. THis is what the getting started should contain.

File: style.css (hunk 1 above)
Comment (chrystalme) [reply style.css]: - [ ] Kindly implement what the @thecodechaser  asked during his review

**Review by chrystalme — CHANGES_REQUESTED** [review]
# Hello @petushka1 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Suggested changes ♻️ 

- [ ] Kindly use a descriptive name for your pull request. `Linters` is not a descriptive name. You can try `Review request for Hello Microverse Project`
- [ ] Also, some of the comments from the previous reviewer were not followed. Kindly use classes for your CSS items.

Cheers and happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Also, you can reach me on [Twitter](https://twitter.com/afam_ifeanyi) or [Slack](https://microverse-community.slack.com/team/U019ZC99KFA)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi @petushka1, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**